bot/bot.py

```diff
@@ -57,30 +72,110 @@ async def register(inter: ApplicationCommandInteraction, nickname: str, password
                   timestamp=datetime.now(timezone))
 
     await inter.edit_original_response(embed=embed)
-    return 
 
-@bot.slash_command(name="unregister", description="Unregister a nickname with a password")
-async def unregister(inter: ApplicationCommandInteraction, nickname: str, password: str) -> None:
+    await login(inter, nickname, password)
+    return
+
+@bot.slash_command(name="login", description="Login to your account")
+async def login(inter: ApplicationCommandInteraction, nickname: str, password: str) -> None:
     
+    await inter.response.defer(ephemeral=True)
+    if not await database.is_password_and_nickname_valid(nickname, password, str(inter.author.id)):
+        embed = Embed(title="Confession",
+                    description="Sorry your nickname and password does not match in the database",
+                    timestamp=datetime.now(timezone))
+        await inter.edit_original_response(embed=embed)
+        return
+
+    await database.logout(str(inter.author.id))
+
+    embed = Embed(title="Confession",
+                description=f"Successfully logged in as `{nickname}`",
+                timestamp=datetime.now(timezone))
+
+    await database.login(nickname, password, str(inter.author.id))
+
+    await inter.edit_original_response(embed=embed)
+    return
+
+@bot.slash_command(name="logout", description="Logout from current account")
+async def logout(inter: ApplicationCommandInteraction) -> None:
+        
         await inter.response.defer(ephemeral=True)
-        if not await database.is_password_and_nickname_valid(nickname, password, str(inter.author.id)):
-            embed = Embed(title="Confession",
-                        description=f"Sorry, the nickname: `{nickname}`, does not exist in the database.",
-                        timestamp=datetime.now(timezone))
-            await inter.edit_original_response(embed=embed)
-            return 
-    
-        await database.unregister_nickname(nickname, password, str(inter.author.id))
+        valid = await database.logout(str(inter.author.id))
+        if valid is False:
+            embed = Embed(title="Error", description="You are not logged in.", color=0xFF0000)
+            await inter.edit_original_message(embed=embed)
+            return
+        
         embed = Embed(title="Confession",
-                    description=f"RIP 🥀 {nickname}\n\nFly high butterfly 🥹\n\n🕊️ Died peacefully in {datetime.now(timezone).strftime('%b %d, %Y')}")
+                    description="Successfully logged out",
+                    timestamp=datetime.now(timezone))
     
-        await inter.channel.send(embed=embed)
+        await inter.edit_original_response(embed=embed)
+        return
+
+
+@bot.slash_command()
+async def user(inter):
```

Comment:
Walang types?

Reply:
wala nga ata dapat param eh

Reply:
https://docs.disnake.dev/en/stable/ext/commands/slash_commands.html#subcommands-and-groups